### ISSUE TITLE
mvn validate -> bash: mvn: command not found

### ISSUE THREAD

**Issue by Rubenkr3** [amzn/alexa-avs-raspberry-pi#160]
In point 9. Start the Client (build the app) I have to type `mvn validate`.
The awnser is `bash: mvn: command not found.` 
I don't know what to do, can anybody help me? 

Thanks in advance!

**Comment by mansong1** [comment]
Do you have maven installed? If so then try running with sudo: _sudo mvn validate_.

**Comment by Rubenkr3** [comment]
I saw that I forgott to tell the shell were to find maven. So I did it now but the problem remains. But allready the command `mvn -version` is not working. (Same as bevor) :/

**Comment by cwilkes** [comment]
@Rubenkr3 can you type `echo $PATH` and see if there's something in there that looks like it is the directory that has the `mvn` executable?

**Comment by sparkgene** [comment]
@Rubenkr3 did you Logout and Login after creating `/etc/profile.d/maven.sh`
or
type the command on your terminal
```
export M2_HOME=/opt/apache-maven-3.3.9
export PATH=$PATH:$M2_HOME/bin
```

**Comment by ajotwani** [comment]
@Rubenkr3 Can you please confirm if you're still having this issues?

**Comment by Rubenkr3** [comment]
@cwilkes if I type `echo $PATH` there is nothing with `mvn` in it.

@sparkgene I only Login oder Logout on putty and VNC to control my Raspi. The Raspi itself is running without contact.

@ajotwani I restarted the project but I still have this problem. Also in the 2nd try.

**Comment by ishc** [comment]
Having the same issue here.

**Comment by keleric** [comment]
What is the output of 'which mvn'? If it's empty, then your path doesn't know about maven. How/Where did you install it?

**Comment by aprilsnows** [comment]
@sparkgene  Thanks. I also had problem like this before. But after I trying sparkgene's solution, it works. (type the command on your terminal).

**Comment by ajotwani** [comment]
Closing as resolved.

**Comment by Jai008** [comment]
I have same issue, if I give command echo $PATH then details displayed and also i tried export commands to set variables still facing the issue. can any body plz help me

**Issue:** if I type command 'mvn clean install' error displayed as "bash: mvn: command not found"

Tried to resolve using below commands 
export M2_HOME=/opt/apache-maven-3.3.3
export PATH=$PATH:$M2_HOME/bin

**Comment by jimbroze** [comment]
I am now having the same error when trying to run the sample app with "mvn exec:exec"

**Comment by dleybz** [comment]
Same error as jimbroze. Is mvn no longer pre-installed with Jessie maybe?

**Comment by mayuresh23sawant** [comment]
Even I am having the same problem. Can anyone help please?

**Comment by keleric** [comment]
automated_install.sh should install maven. If it didn't, it's possible your connection to the internet failed while it was trying to download it. I would suggest rerunning automated_install.sh and verifying that maven was installed correctly.

**Comment by testack18** [comment]
I run into the same issue. After a few search I found this site and checked the automated_install.sh to see what exact command is used to install maven. When I run it by hand everything worked.

BUT: question -> how can I know if everything else is now installed? Since I started troubleshooting my terminal overwritten the output so I cannot see the install result. Does the installer has an install.log where I can review the installation result later for troubleshooting?
How can I make sure if everything else is now correctly installed? Can I simple re-run the installer once or more times?
